Clustering/src/Driver.java

```diff
@@ -72,9 +72,27 @@ public static void main(String[] args){
      * @return true if arguments are valid
      */
     private static boolean verifyArgs(String[] args){
-        /**
-         * TODO
-         */
+        String boi = args[0];
+        if(!args[0].equals("aco") && !args[0].equals("dbs") && !args[0].equals("km") && !args[0].equals("nn") && !args[0].equals("pso")){
+            System.out.println("The first argument selects the algorithm");
+            System.out.println("Your input for the first argument was:\t" + args[0]);
+            System.out.println("Valid choices are: \n\taco\n\tdbs\n\tkm\n\tnn\n\tpso");
```

Comment:
Instead of printing 3 lines. I would just print 1 or 2 that says:   
"Error in first argument (algorithm). Correct uses are:..." and list the correct usage